src/workerd/server/workerd.c++

```diff
@@ -1283,10 +1283,15 @@ private:
     kj::StringPtr message = "Noticed configuration change, reloading shortly...\r";
     kj::FdOutputStream(STDERR_FILENO).write(message.begin(), message.size());
 
+    static auto const waitForIt = [](kj::Promise<void> promise,
```

Comment:
Nitpick: better name for this as it's not immediately clear why it's necessary just from the name, maybe `setPromiseResult` or `mapPromiseResult`?